### PR TITLE
Adding in function for checking the time until the resetting of crates.

### PR DESCRIPTION
**Pull request by rfukutome**

### CODE REVIEW

File: message_handler.py [hunk 1]
```diff
@@ -46,6 +48,23 @@ def handle_message_invalid():
     '''
     return 'I don\'t understand your accent. Try !help'
 
+def handle_message_crate_time():
+    # Get current UTC time (UTC is common for world).
+    current_utc_time = datetime.datetime.utcnow()
+
+    # Turn UTC time to correct time zone (-7).
+    current_timezone_time = current_utc_time + datetime.timedelta(hours=-7)
```
Comment (TylerNagamine) [inline message_handler.py:56]: Can python do timezone with daylight savings?

File: message_handler.py [hunk 2]
```diff
@@ -46,6 +48,23 @@ def handle_message_invalid():
...
+    
+    # Find the difference in the corrected time and Sunday at 6PM.
+    next_crate_time = current_timezone_time.replace(hour=18, minute=0, second=0)
```
Comment (TylerNagamine) [inline message_handler.py:59]: Or maybe better to just do utc here

**Review by TylerNagamine — APPROVED** [review]

File: message_handler.py (hunk 2 above)
Comment (annapeskin) [reply message_handler.py:59]: If datetime supports UTC only, everything needs to be converted to UTC. No need to convert to PST

File: message_handler.py (hunk 1 above)
Comment (annapeskin) [reply message_handler.py:56]: Need to add pydoc type comments here

File: message_handler.py (hunk 2 above)
Comment (annapeskin) [reply message_handler.py:59]: Also need to explain the negative time thing.. comment is ambiguous

**Comment by annapeskin** [comment]
TODO: add this to the dict that holds all of the handle functions, otherwise it will never actually get called.

**Pull request merged**